committee-steering/governance/sig-governance.md

```diff
@@ -1,62 +1,135 @@
 # SIG Roles and Organizational Governance
 
-This charter adheres to the conventions described in the [Kubernetes Charter README]. It will be updated as needed to meet the current needs of the Kubernetes project.
+This charter adheres to the conventions described in the [Kubernetes Charter README].
+It will be updated as needed to meet the current needs of the Kubernetes project.
 
-In order to standardize Special Interest Group efforts, create maximum transparency, and route contributors to the appropriate SIG, SIGs should follow these guidelines:
+In order to standardize Special Interest Group efforts, create maximum
+transparency, and route contributors to the appropriate SIG, SIGs should follow
+these guidelines:
 
 - Create a charter and have it approved according to the [SIG charter process]
-- Meet regularly, at least for 30 minutes every 3 weeks, except November and December
-- Keep up-to-date meeting notes, linked from the SIG's page in the community repo
-- Record meetings and make them publicly available on the [Kubernetes Community YouTube playlist]
-- Report activity with the community via the kubernetes-dev mailing list at least once a quarter. Whichever format the SIG uses should _always_ be reported to the kubernetes-dev mailing list as this is the list the community depends on for SIG updates.  
-  - Each SIG is assigned an update during the monthly community meeting throughout the year. The meeting host will publish the notes to the kubernetes-dev mailing list with the update.  
-  - If a SIG is doing an update during KubeCon + CloudNativeCon, the SIG Chairs should ensure that a link to the recording of the video is sent to the kubernetes-dev mailing list.  
-  - YouTube presentation or mailing list update sent to the kubernetes-dev mailing list.
-- Participate in release planning meetings and retrospectives, and burndown meetings, as needed
-- Ensure related work happens in a project-owned github org and repository, with code and tests explicitly owned and supported by the SIG, including issue triage, PR reviews, test-failure response, bug fixes, etc.
-- Use the [forums provided] as the primary means of working, communicating, and collaborating, as opposed to private emails and meetings
-
-The process for setting up a SIG or Working Group (WG) is listed in the [sig-wg-lifecycle] document.
+- Meet regularly, at least for 30 minutes every 3 weeks, except November and
+December
+- Keep up-to-date meeting notes, linked from the SIG's page in the community
+repo
+- Ensure other SIGs and community members can find notes of meetings,
+discussions, designs, and decisions
+- Record meetings and make them publicly available on the
+[Kubernetes Community YouTube playlist]
+- Report activity with the community via the kubernetes-dev mailing list at
+least once a quarter. Whichever format the SIG uses should _always_ be reported
+to the kubernetes-dev mailing list as this is the list the community depends on
+for SIG updates.  
```

Comment:
this was the new process that was PR'ed in separately from Jorge when we went to monthly vs weekly cc/ @castrojo

Reply:
We've been having the host of the meeting posting the full notes directly to -dev after each meeting now which does include the SIG updates.  In hindsight the wording here makes it sound like you'd be getting the updates from the SIGs when we really mean that we're doing a much better job of getting the full meeting notes out to -dev regularly. 

Since moving to the monthly format we have not yet had a SIG need to use the "post your youtube video to the list" or do any of the other extra methods needed to post to -dev. We mostly added these as options in the case SIGs would need to be more flexible from missing a community meeting.

Reply:
> Since moving to the monthly format we have not yet had a SIG need to use the "post your youtube video to the list" or do any of the other extra methods needed to post to -dev. We mostly added these as options in the case SIGs would need to be more flexible from missing a community meeting.

Thanks, this helps clarify! Maybe we could explicitly state here that point 2 and 3 are optional?

Reply:
PTAL 

- Report activity with the community via the kubernetes-dev mailing list at
least once a quarter. Whichever format the SIG uses should _always_ be reported
to the kubernetes-dev mailing list as this is the list the community depends on
for SIG updates.  
  - Each SIG is assigned an update during the monthly community meeting
  throughout the year. The meeting host will publish the notes to the
  kubernetes-dev mailing list with the update.  
  - Due to limited opportunities (twice a year) at the community meeting to
  present, your SIG will need to figure out two other options to deliver on SIG 
  updates:
    1. During KubeCon + CloudNativeCon or  
    2. YouTube presentation or mailing list update sent to the kubernetes-dev
  mailing list.